frontend/src/utils/fetchActivities.tsx

```diff
@@ -0,0 +1,44 @@
+import { ActivityProps } from "@/types";
+
+export const fetchActivities = async (
+  latitude: number,
+  longitude: number,
+  radius: number,
+) => {
+  const GOOGLE_PLACES_API_KEY =
+    process.env.EXPO_PUBLIC_GOOGLE_PLACES_API_KEY || "undefined";
+
+  const getImageUrl = (photos: any, apiKey: string) => {
+    if (photos && photos.length > 0) {
+      const photoReference = photos[0].photo_reference;
+      return `https://maps.googleapis.com/maps/api/place/photo?maxwidth=400&photo_reference=${photoReference}&key=${apiKey}`;
+    }
+    return "https://images.unsplash.com/photo-1506748686214-e9df14d4d9d0";
```

Comment:
No need to change at this time, but we can consider default image uri for each category in the future